docs/usage.md

```diff
@@ -309,7 +312,80 @@ sufficient confirmations, an off-chain program called the
 [vigilante BTC Staking tracker](https://github.com/babylonlabs-io/vigilante)
 notifies the Babylon chain about the staking transaction's inclusion.
 
-## 4. Unbonding Transaction
+## 4. Delegation Expansion
+
+Delegation expansion allows you to extend an existing BTC stake with additional
+finality providers or renew the timelock without going through the full unbonding
+process. This implements RFC 037 BTC Stake Expansion.
+
+The expansion process involves:
+1. Creating an unsigned staking expansion transaction
+2. Registering the expansion on the Babylon chain
+3. Signing and submitting the expansion transaction to Bitcoin
+
+### 4.1 Staking Expansion Registration
+
+First, create the initial expansion registration message that will be signed
+by the Babylon Provider as a Babylon Genesis transaction.
+
+```ts
+const {
+  signedBabylonTx,
+  stakingTx: stakingExpansionTx
+} = await manager.stakingExpansionRegistrationBabylonTransaction(
+  stakerInfo,
+  stakingInput,
+  babylonBtcTipHeight,
+  inputUTXOs,
+  feeRate,
+  babylonAddress,
+  {
+    stakingTx: previousStakingTx,
+    paramVersion: previousStakingParamsVersion,
+    stakingInput: previousStakingInput,
+  }
+);
+```
+
+**Important Notes:**
+- The expansion amount must equal the previous staking amount (increases are
+not yet supported).
+- All finality providers from the previous staking must be included in the
```

Comment:
```suggestion
- The expansion inherits the same set of Finality Providers from the previous staking. No new providers can be added or removed at this stage
```

Reply:
Or i am wrong and we need to include them somewhere? If so who do i query which one i had? Just curious